src/Extension/AccountReset/SecurityAdminExtension.php

```diff
@@ -34,6 +34,10 @@ class SecurityAdminExtension extends Extension
         'reset',
     ];
 
+    private static $url_handlers = [
```

Comment:
We change the SecurityAdmin to be a plain ModelAdmin. A side affect of that seems to be that we can create REST endpoints directly on this controller anymore.

MFA has a "reset" endpoint on Security used to trigger a force reset of a Member MFA settings.

I change that endpoint to be `users/reset/$ID` instead of `reset/$ID`.